docs/spec/server.md

```
@@ -0,0 +1,92 @@
+# サーバーサイド仕様
+
+> **Note**: このファイルは [SPECIFICATION.md](../../SPECIFICATION.md) の6章を分割したものです。
+
+---
+
+## 6.1 ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)